src/components/HdTabsMenu.vue

```diff
@@ -170,7 +170,7 @@ export default {
       width: 24px;
       height: 100%;
       background-color: $primary-bg;
-      background-image: url('~homeday-blocks/src/assets/small-icons/ic_chevron.svg');
+      background-image: url('~homeday-assets/S/chevron.svg');
```

Comment:
That's definitely the best way to do it 👍  I just didn't want to do it in this PR to not stretch its scope ("use homeday-assets instead of the local assets").